scripts/xenadmin-build.sh

```diff
@@ -217,7 +222,33 @@ cp ${REPO}/XenAdmin/bin/Release/{CommandLib.pdb,${BRANDING_BRAND_CONSOLE_NO_SPAC
 
 cd ${OUTPUT_DIR} && zip -r -m  ${BRANDING_BRAND_CONSOLE_NO_SPACE}.Symbols.zip *.pdb
 
-sha256sum ${OUTPUT_DIR}/${BRANDING_BRAND_CONSOLE_NO_SPACE}.msi > ${OUTPUT_DIR}/${BRANDING_BRAND_CONSOLE_NO_SPACE}.msi.checksum
+msi_checksum_with_file_name=`sha256sum ./CitrixHypervisorCenter.msi`
+echo $msi_checksum_with_file_name > ${OUTPUT_DIR}/${BRANDING_BRAND_CONSOLE_NO_SPACE}.msi.checksum
+msi_checksum=`echo $msi_checksum_with_file_name | cut -d " " -f 1`
```

Comment:
FYI, you don't need to change it, but you could also do
`msi_checksum=($msi_checksum_with_file_name)`

Reply:
`msi_checksum` is only for the checksum. The snippet removes the file's name that is returned from the `sha256` call.

```sh
PS C:\Users\danilod> sha256sum.exe .\test.txt
\e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855 *.\\test.txt
```

I tried the following:
```sh
msi_checksum_with_file_name=`sha256sum ./test.sh`
msi_checksum=($msi_checksum_with_file_name)
echo $msi_checksum
```

But it returned this:
```sh
PS C:\Users\danilod> sh .\test.sh
)eba71bcf933510ba998149c20a4fbd9691f6c8573bc534d4c3cb517f2649783d *./test.sh
```

Reply:
EDIT. I was running this on ps. Your version works. I'll update the line